pkg/etcd/etcd.go

```diff
@@ -1011,6 +1023,22 @@ func (e *ETCD) RemovePeer(ctx context.Context, name, address string, allowSelfRe
 	return nil
 }
 
+// Creates a specific k8s client for condition patches
+func (e *ETCD) setK8sClient() error {
```

Comment:
Since this is just used in one place, I think rather than adding a field to the struct, we should just create a variable in `manageLearners` that is passed into `setEtcdStatusCondition`